changelogs/unreleased/gh-5568-readonly-reason.md

```diff
@@ -0,0 +1,24 @@
+## feature/core
+
+* Error objects with the code `box.error.READONLY` now have additional fields
+  explaining why the error happened (gh-5568).
+
+  For instance, if the error was due to `box.cfg{read_only = true}`, then the
+  error will have fields:
+  ```
+  {
+    reason = 'state',
+    state = 'read_only'
+  }
+  ```
+  Or if it was due to not being a leader with `box.cfg{election_mode = ...}` set
+  not to `off`, then:
+  ```
+  {
+    state: <box.info.election.state of this instance>,
+    leader_id: <box.info.id of the leader>,
+    leader_uuid: <box.info.uuid of the leader>,
+    reason: election,
+    term: <box.info.election.term of this instance>
+  }
+  ```
```

Comment:
IMO no need to go into details here - the changelog should be brief while all the details should go to the documentation. I'd leave only the first two lines and delete the rest.

Reply:
Ok, I don't mind dropping it.
<details><summary>diff</summary>

```Diff
diff --git a/changelogs/unreleased/gh-5568-readonly-reason.md b/changelogs/unreleased/gh-5568-readonly-reason.md
index 89a5f1d68..f3a2db986 100644
--- a/changelogs/unreleased/gh-5568-readonly-reason.md
+++ b/changelogs/unreleased/gh-5568-readonly-reason.md
@@ -2,23 +2,3 @@
 
 * Error objects with the code `box.error.READONLY` now have additional fields
   explaining why the error happened (gh-5568).
-
-  For instance, if the error was due to `box.cfg{read_only = true}`, then the
-  error will have fields:
-  ```
-  {
-    reason = 'state',
-    state = 'read_only'
-  }
-  ```
-  Or if it was due to not being a leader with `box.cfg{election_mode = ...}` set
-  not to `off`, then:
-  ```
-  {
-    state: <box.info.election.state of this instance>,
-    leader_id: <box.info.id of the leader>,
-    leader_uuid: <box.info.uuid of the leader>,
-    reason: election,
-    term: <box.info.election.term of this instance>
-  }
-  ```
```
</details>